dist/app.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
```

Comment:
Are you sure you want to commit your `dist/` folder? Typically would be generated in a build step on deploy..

Reply:
oof, no, I should add that to my .gitignore, right?  This is my first time deploying a back end.

Reply:
Yep! Should be able to just add `dist/` as a new line and it should start ignoring them